config.go

```diff
@@ -29,6 +29,8 @@ import (
 
 	"github.com/golang/glog"
 
+	"strings"
```

Comment:
Should be at the top where all the golang-sdk import are

Reply:
That's the order `goimports` suggests. Anyway I manually did it.

Reply:
Thats because strings was in a separate block. goimports sorts block-wise :)

Reply:
Somebody should fork goimports to implement the kube import order conventions. k8s.io/kimport.